src/TestRunner.js

```diff
@@ -475,7 +475,16 @@ class TestRunner {
     var config = this._config;
     if (!reporter) {
       var TestReporter = require(config.testReporter);
-      reporter = new TestReporter();
+      if (config.useStderr) {
+        reporter = new TestReporter(
+          Object.create(
+            process,
+            { stdout: { value: process.stderr } }
```

Comment:
meh, I know you love this style but I'd really prefer this without the spaces. It makes it just slightly easier to turn this into a multi-line object expression later if we need to. #bikeshed #codestyle